README.md

```diff
@@ -126,13 +126,34 @@ merge:
   # "rebase", "squash", and "ff-only".
   method: squash
 
-  # Allows the merge method that is used when auto-merging a PR to be different based on the
+  ##### branch_method has been DEPRECATED in favor of merge_method #####
+  # 
+  # Allows the merge method that is used when auto-merging a PR to be different
   # target branch. The keys of the hash are the target branch name, and the values are the merge method that
   # will be used for PRs targeting that branch. The valid values are the same as for the "method" key.
   # Note: If the target branch does not match any of the specified keys, the "method" key is used instead.
   branch_method:
     develop: squash
     master: merge
+  ##### branch_method has been DEPRECATED in favor of merge_method #####
+
+  # Allows the merge method that is used when auto-merging a PR to be different
+  # based on trigger criteria. The first method where ALL triggers match will
+  # be used. Otherwise, the method specified previously in merge/method will be # used.
```

Comment:
I usually use quotes and dots to refer to specific keys: `"merge.method"` or just `"method"` as in the `branch_method` docs. Also looks like the last word should be on a new line (or delete the `#`)